### ISSUE TITLE
Add "make check" target

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#1439]
The virtual standard for invoking the test suite of `make` based build systems is `make check`. We also should support that. Multiple ways to do that:
* could be analias for e.g. `testinstall`
* could simply print a list of the actual test targets and what they do
* ...?

**Comment by ChrisJefferson** [comment]
Let's just link it to `testinstall`, that seems like a sensible value.

**Comment by fingolfin** [comment]
I made PR #3285 to resolve this, as I was again asked by people why `make check` does not work, and I agree that we should have it, as it is the "industry standard" and it makes sense to allow people to do things the "usual" way.

But I decided against making this an alias for `make testinstall`, as I never use `make testinstall` myself; for starters, it runs the suite twice, and thus takes twice longer; it creates log files I don't want; and worst of all, it does not set a proper exit code, so there is no good way to test programatically if the tests actually passed (yes, one can grep through the output; but note that I said "there is no good way", not: "there is now way" ;-).

So instead, I simply made `make check` run `$(TESTGAP) tst/testinstall.g`, which does the job nicely, and the resulting target description is much easier to understand than the code for the `testinstall` target.
